core/repository/block/pg.ts

```diff
@@ -133,7 +135,8 @@ class BlockPGRepo implements IBlockPGRepository {
         return rawBlockIds.map(block => block.id);
     }
 
-    async getMany(limit: number, offset: number): Promise<Array<Block>> {
+    async getMany(offset: number = 0, limit: number = 0): Promise<Array<Block>> {
```

Comment:
Why did you switch parameters?

Reply:
To suit memory repository, and imho better sequence

Reply:
Better will be general. In general u say limit offset. Without limit request not possible, but without offset possible. Right order `limit, offset`. I fixed it in previous merge. Return to right order please.